src/dialect/sqlite.rs

```diff
@@ -35,6 +35,10 @@ impl Dialect for SQLiteDialect {
             || ('\u{007f}'..='\u{ffff}').contains(&ch)
     }
 
+    fn supports_filter_during_aggregation(&self) -> bool {
```

Comment:
This PR supports filtering for any function with an `OVER` clause, correct? Thus, this name is no longer quite right I don't think (again maybe this is something we change in a follow on PR) 🤔

Reply:
Yes, let's look at that in a separate pr. This may need some investigation to make sure we do not remove useful functionality